### ISSUE TITLE
Slider border too tall with no text

### ISSUE THREAD

**Issue by celiasmith** [nengo/nengo_viz#243]
The border around the slider looks odd when you remove labels, because it's sized to have them there.

Also, the text labels should maybe be truncated or '...' to match the width of the slider?  They can run very far over at the moment.

**Comment by jgosmann** [comment]
Yeah, the CSS style `text-overflow: ellipsis;` should be set on those labels and we have to make sure that their size is limited.

Adjusting the border could be non-trivial because the label is at the top. If we set it to `display:none;` the sliders would suddenly move up. Thus, hiding would have to move the whole component by the same amount down (which is an `em` value, so we have to get fhe pixel value first ...).

**Comment by tcstewar** [comment]
This should probably be kept in mind in the TypeScript refactoring, as I think it's a more general problem than just for the sliders.  The same thing happens to the other plots as well. 

That said, the ellipsis thing would probably be a nice quick thing to try out.
